src/olympia/reviewers/views.py

```diff
@@ -755,7 +755,8 @@ def review(request, addon, channel=None):
                 # a ton of potentially duplicate queries with all the useless
                 # Addon transforms.
                 Prefetch(
-                    'addon', queryset=Addon.objects.all().only_translations())
+                    'addon',
+                    queryset=Addon.unfiltered.all().only_translations())
```

Comment:
We need a test that proves this change works.  Something in `reviewers/tests/test_views.py` `TestReview` - If you look at the various `test_abuse_reports` methods that test scenarios you should be able to copy one and change it to work on a deleted addon instead and check the abuse reports section still works.